kerascv/layers/anchor_generators/multi_scale_anchor_generator.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020 The Keras CV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from .anchor_generator import AnchorGenerator
+
+
+class MultiScaleAnchorGenerator(tf.keras.layers.Layer):
```

Comment:
Given the similarity with AnchorGenerator why not subclass it?

Reply:
This is the class that accepts a list of things and then apply to each one of the anchor generator?